Spendy/AddReminderViewController.swift

```diff
@@ -34,9 +36,19 @@ class AddReminderViewController: UIViewController, TimeCellDelegate {
         
         addGestures()
         
+        formatter = NSDateFormatter()
```

Comment:
instead of creating a new NSDateFormatter instance each time (it is a little expensive), let's just use a static `Helper.sharedInstance.dateFormatter`